metrics/resource_view.go

```diff
@@ -186,6 +226,7 @@ func ClearMetersForTest() {
 		meter.m.Stop()
 		delete(allMeters.meters, k)
 	}
+	cleanupOnce = new(sync.Once)
```

Comment:
Hmm -- this will clear the the `Once`, but won't the old goroutine still hang around?

Reply:
Yes, I realized that once I noticed that `go test --race` was failing.

I've still got a bit of ramping up to do on concurrency in go...  I'll